proto/svc-storage-grpc-parcel.proto

```diff
@@ -56,8 +56,11 @@ enum ParcelStatus {
 
 // Data struct with parcel data
 message Data {
-    // the itinerary UUID associated with the parcel
-    string itinerary_id = 1;
+    // the user UUID associated with this parcel
+    string user_id = 1;
+
+    // the weight (in grams)
+    uint32 weight_g = 2;
```

Comment:
We decided to go with the full unit name so there can be no confusion as to what the abbreviation means. So should be named `weight_grams`

Reply:
https://github.com/Arrow-air/svc-storage/pull/76/commits/73d3136188511e164c0e28f52e3e373b1e657558